consensus/src/sync/peer_list.rs

```diff
@@ -58,22 +60,27 @@ impl<N: Network> Default for PeerList<N> {
         Self {
             peers_set: Default::default(),
             peers: Default::default(),
+            notify_nonempty: Default::default(),
         }
     }
 }
 
 impl<N: Network> Clone for PeerList<N> {
-    fn clone(&self) -> Self {
-        Self {
+    fn clone(&self) -> PeerList<N> {
```

Comment:
Why can't both lines be `Self`?

Reply:
Can both be `Self`. I deleted the implementation and recreated it during the creation of the PR, so I used the more explicit style. Interesting that it still ended up having such a small diff. I can change it to `Self` if you want.

Reply:
It's nitpicking but by having `Self` it can stay `Self`, even if the name of the struct changes :)

Reply:
Changed it.